WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Welcome/MigrationWelcomeViewController.swift

```diff
@@ -20,13 +22,22 @@ final class MigrationWelcomeViewController: UIViewController {
     private lazy var bottomSheet: MigrationActionsView = {
         let actionsView = MigrationActionsView(configuration: viewModel.configuration.actionsConfiguration)
         actionsView.translatesAutoresizingMaskIntoConstraints = false
+        actionsView.primaryHandler = { [weak self] configuration in
+            self?.tracker.track(.welcomeScreenContinueTapped)
+            configuration.primaryHandler?()
+        }
+        actionsView.secondaryHandler = { [weak self] configuration in
+            self?.tracker.track(.welcomeScreenHelpButtonTapped)
+            configuration.secondaryHandler?()
+        }
         return actionsView
```

Comment:
Prior to these changes, the events tracking logic was executed inside `configuration.primaryHandler` and `configuration.secondaryHandler`.

But since both the "Need help?" and "Me" button call `configuration.secondaryHandler?()`, it wasn't possible to differentiate between these 2 events:

```
migration_welcome_screen_help_button_tapped
migration_welcome_screen_avatar_tapped
```